presto-product-tests/conf/docker/files/apply-site-xml-override.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
```

Comment:
would it be possible to check if the setting is correctly applied in hive? Maybe some Hive SQL command?
Maybe create table in Hive?

Reply:
added